roles/mongodb/tasks/mongodb_rhel.yaml

```diff
@@ -0,0 +1,34 @@
+---
+- include: mongodb_rhel6_dependencies.yaml
+  when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '6'
+
+- name: Add mongodb key
+  become: yes
+  rpm_key:
+    key: https://www.mongodb.org/static/pgp/server-3.2.asc
+    state: present
+  tags: [databases, mongodb]
+
+- name: Add mongodb repository
+  become: yes
+  yum_repository:
+    name: mongodb-org-3.2
+    description: MongoDB Repository
+    gpgcheck: yes
+    enabled: yes
+    baseurl: https://repo.mongodb.org/yum/redhat/{{ ansible_distribution_major_version }}Server/mongodb-org/3.2/x86_64/
+    gpgkey: https://www.mongodb.org/static/pgp/server-3.2.asc
```

Comment:
Is this what we want: https://github.com/StackStorm/ansible-st2/pull/65/files#diff-53ff4fa5f01506a4719fead10b74605bR17 ?